src/Microsoft.ML.CpuMath/Microsoft.ML.CpuMath.csproj

```diff
@@ -28,6 +28,7 @@
     <Compile Remove="CpuMathUtils.netcoreapp.cs" />
     <Compile Remove="SseIntrinsics.cs" />
     <Compile Remove="AvxIntrinsics.cs" />
+    <PackageReference Include="System.Memory" Version="$(SystemMemoryVersion)" />
```

Comment:
nit: Having different items in their own ItemGroups makes the file easier to manually parse.

Reply:
I'll add a blank line above this line to segregate the Compile and PackageReference groups. I added this to the same ItemGroup to limit the number of `TargetFramework` Conditions.